assets/js/googlesitekit/datastore/user/permissions.js

```diff
@@ -239,6 +239,14 @@ const baseSelectors = {
 		// shareable and the user has the "read shared module data"
 		// capability for.
 		return Object.values( modules ).reduce( ( slugs, module ) => {
+			if ( module.slug === 'analytics' && isGA4DashboardView ) {
+				return slugs;
+			}
+
+			if ( module.slug === 'analytics-4' && ! isGA4DashboardView ) {
+				return slugs;
+			}
```

Comment:
I know this is how I formatted the code in my PoC, but I would really prefer to see these conditions combined (or at least use an `else` to make it clear from a quick scan that the conditions are mutually exclusive). What do you think?

```suggestion
			if (
				( module.slug === 'analytics' && isGA4DashboardView ) ||
				( module.slug === 'analytics-4' && ! isGA4DashboardView )
			) {
				return slugs;
			}
```

Reply:
SGTM 👍

Reply:
Updated ✅